recipes/b2/standard/conanfile.py

```diff
@@ -0,0 +1,94 @@
+from conans import ConanFile, tools
+import os
+
+
+class B2Conan(ConanFile):
+    name = "b2"
+    homepage = "https://boostorg.github.io/build/"
+    description = "B2 makes it easy to build C++ projects, everywhere."
+    topics = ("conan", "installer", "boost", "builder")
+    license = "BSL-1.0"
+    settings = "os_build", "arch_build"
+    url = "https://github.com/conan-io/conan-center-index"
+
+    '''
+    * use_cxx_env: False, True
+
+    Indicates if the build will use the CXX and
+    CXXFLAGS environment variables. The common use is to add additional flags
+    for building on specific platforms or for additional optimization options.
+
+    * toolset: 'auto', 'cxx', 'cross-cxx',
+    'acc', 'borland', 'clang', 'como', 'gcc-nocygwin', 'gcc',
+    'intel-darwin', 'intel-linux', 'intel-win32', 'kcc', 'kylix',
+    'mingw', 'mipspro', 'pathscale', 'pgi', 'qcc', 'sun', 'sunpro',
+    'tru64cxx', 'vacpp', 'vc12', 'vc14', 'vc141', 'vc142'
+
+    Specifies the toolset to use for building. The default of 'auto' detects
+    a usable compiler for building and should be preferred. The 'cxx' toolset
+    uses the 'CXX' and 'CXXFLAGS' solely for building. Using the 'cxx'
+    toolset will also turn on the 'use_cxx_env' option. And the 'cross-cxx'
+    toolset uses the 'BUILD_CXX' and 'BUILD_CXXFLAGS' vars. This frees the
+    'CXX' and 'CXXFLAGS' variables for use in subprocesses.
+    '''
+    options = {
+        'use_cxx_env': [False, True],
+        'toolset': [
+            'auto', 'cxx', 'cross-cxx',
+            'acc', 'borland', 'clang', 'como', 'gcc-nocygwin', 'gcc',
+            'intel-darwin', 'intel-linux', 'intel-win32', 'kcc', 'kylix',
+            'mingw', 'mipspro', 'pathscale', 'pgi', 'qcc', 'sun', 'sunpro',
+            'tru64cxx', 'vacpp', 'vc12', 'vc14', 'vc141', 'vc142' ]
+    }
```

Comment:
I think this options should be removed in the package_id() method as those only inject different flags to the b2 command line, but do not generate a different binary for b2. In other words, those options only affect the compilation of the consumers